test-crawler/integrations_tests/01_crawl_folder_build_catalogue.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+go run . -crawl=src
+find "outputs" -type f -exec echo Found file {} \;
+# for FILE in "outputs/*"; do echo $FILE | cat $FILE; done
```

Comment:
I ran the test, and in my opinion, this output is unnecessary - the terminal is filled with useless logging lines. I would remove these.